lib/beaker-hostgenerator/parser.rb

```diff
@@ -14,7 +14,28 @@ module BeakerHostGenerator
   # each token you would call `is_ostype_token?` and/or `parse_node_info_token`.
   module Parser
 
-    # Capture role and bit width information about the node.
+    # Parses a single node definition into the following components:
+    #
+    #   * bits             Uppercase-only alphanumeric
+    #                      Examples: 32, 64, POWER7 (note 7POWER is invalid)
```

Comment:
Done in https://github.com/puppetlabs/beaker-hostgenerator/pull/32/commits/1ecf4d1135662cc44b39a749e044fa6a7e2d4413